### ISSUE TITLE
[Bug]: Xray bug

### ISSUE THREAD

**Issue by sfemnksbk3** [PluginBugs/Issues-ItemsAdder#1233]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-ItemsAdder/issues) to check if the same issue was already reported.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if a solution is already known.
- [X] I already asked on the **#💬ia-community-help** channel on **Discord** to know if anyone already has a solution for the issue.

### What happened?

There is also no such problem in the old version, and when the current version also breaks the 4x4, the xray bug.

### Steps to reproduce the issue

1. Get demoniac hammer
2. Break block 4x4 and xray bug

### Server version

This server is running Purpur version git-Purpur-1171 (MC: 1.16.5) (Implementing API version 1.16.5-R0.1-SNAPSHOT)

### ItemsAdder Version

ItemsAdder version 2.4.17

### ProtocolLib Version

ProtocolLib version 4.7.10

### LoneLibs Version

LoneLibs version 1.0.14

### FULL server log

No Log

### Screenshots/Videos

Get demoniac hammer break block.

ItemsAdder version 2.4.17
-------------------------------
https://i.ibb.co/WHySH0g/2021-09-27-04-34-04.png

ItemsAdder version 2.3.9
-------------------------------
https://ibb.co/2WwnWxz

Video:
https://youtu.be/EZ9KMJTtK38

**Comment by LoneDev6** [comment]
The ores you see are fake ores.
Paper and Spigot have a protection called anti xray which creates fake blocks around the player.

The ItemsAdder hammer doesn't trigger the block updates around the player to avoid possible lag, so Paper doesn't know that the player has broken the blocks and can't send block updates packet to the player (the hammer digs a lot of blocks each time, so I can't really get rid of this optimization because it would cause lag if a lot of players use the hammers).

Basically the main reason why you see this is because of optimization of the plugin, the downside is that sometimes you will see these glitched blocks (which are not real, they are fake).

I already got a similar report https://github.com/PluginBugs/Issues-ItemsAdder/issues/289#issuecomment-647538584 .

I will try to find a workaround for this situation while still keeping the plugin performance at max without triggering the block updates.

Thanks for the patience
